version.sbt

```diff
@@ -0,0 +1 @@
+version in ThisBuild := "0.1.0-SNAPSHOT"
```

Comment:
Can't we use auto versioning from git?

Reply:
In maven central there is a difference between snapshot and release versions (and therefore repositories) which is by convention part of the version, `sbt-release` is using this `version.sbt` to update the version during the (manual) release. This is probably related to your comment of having releases automatically, which would make this obsolete if you would always use the commit hash as the release version number for example. This would probably need a different configuration in the release plugin though.